src/routes/posts.js

```diff
@@ -33,6 +34,49 @@ const createPost = async (req, res, next) => {
     }
 }
 
+// GET REQUESTS
+
+// GET POST BY ID
+
+const getPost = async (req, res, next) => {
+    const { id } = req.params
+    try {
+        const posts = await db.query(
+            'SELECT p.*, jsonb_agg(c.* ORDER BY c."createdAt" DESC) as comments FROM posts p LEFT JOIN comments c ON p.id = c."postId" WHERE p.id=$1 GROUP BY p.id;',
```

Comment:
@beblicarl  can we use a backtick(`) to do a multiline query here?

something like

```
`
SELECT p.*, jsonb_agg(c.* ORDER BY c."createdAt" DESC) as comments
FROM posts p 
LEFT JOIN comments c ON p.id = c."postId"
WHERE p.id=$1
GROUP BY p.id;

`
```

this is to make it a bit more readable by making the keywords start each line

Reply:
alright